.github/workflows/cml-561.yml

```
@@ -0,0 +1,65 @@
+name: cml#561 mounting cloud object storage with FUSE
+on:
+  schedule:
+    - cron: '45 14 * * 1' # Mondays 1445 UTC
+  push:
+permissions:
+  id-token: write
+jobs:
+  aws:
+    environment: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::342840881361:role/SandboxUser
+          aws-region: us-west-1
+      - run: |
+          sudo apt update
+      - run: |
+          sudo apt install fuse s3fs
+      - run: |
+          sudo mkdir /mnt/objects
+      - run: |
+          tee "$(mktemp)" <<< "$AWS_ACCESS_KEY_ID:$AWS_SECRET_ACCESS_KEY" > /dev/null
+          sudo s3fs -o passwd_file="$_" "$AWS_S3_BUCKET" /mnt/objects
+        env:
+          AWS_ACCESS_KEY_ID: ${{ env.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ env.AWS_SECRET_ACCESS_KEY }}
+          AWS_S3_BUCKET: workshopdvcdata	
```

`-Wtrailing-whitespace`

```suggestion
          AWS_S3_BUCKET: workshopdvcdata
```

---

Thanks! 🦅 👁️